### PR TITLE
Progress on #8

### PR DESCRIPTION
**Pull request by rasmuserik**
parse query parameter through, and render work, based on parameter.

### CODE REVIEW

File: lib/util.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+export function isClient() {
+  return typeof window !== 'undefined'
+}
+
+export function parseQuery(q) {
```
Comment (jacobandresen) [inline lib/util.js]: maybe you could use "querystring" instead? https://nodejs.org/api/querystring.html

File: components/toolbar.js [hunk 2]
```diff
@@ -23,7 +23,10 @@ export default ({title, showMenu, toggleMenu}) => (
       <Link href='/bibdata'>
       <Button flat style={{width: '100%'}} label="Work info" />
     </Link>
-      <Link href='/about'>
+    <Link href='/bibdata?id=ting:1234'>
+      <Button flat style={{width: '100%'}} label="Bibdata - test"  />
```
Comment (jacobandresen) [inline components/toolbar.js]: can we avoid inline styling?  my experience shows that this is "go" or "no go" .. haha

Comment (rasmuserik) [reply components/toolbar.js]: yes, current version is just a quick hack, - which is why it was there :)

File: lib/util.js (hunk 1 above)
Comment (rasmuserik) [reply lib/util.js]: yes, makes sense - just checked the source and it doesn't depends on other (node) modules so it is ok to include in the browser, - the browser-version is not well-maintained though, but the API is npm-stable, so all fine :)

**Pull request closed without merging**